scripts/mock_sensors.py

```diff
@@ -74,7 +74,7 @@ def send_sensor_data(device, SID, channel):
 
             for i in range(len(SID[device])):
                 msg = can.Message(arbitration_id=SID[device][i], dlc=0x8, data=[
-                                  50, 50, 50, 50, 50, 50, 50, 50])
+                                  50, 50, 50, 50, 50, 50, 50, 50], extended_ID=False)
```

Comment:
This extended_ID still somehow works despite its capitalisation but can change as needed